src/components/Pages/Collection/index.js

```diff
@@ -107,7 +107,7 @@ const Collection = ({ id, location }) => {
         `
     } else {
       query = `mutation {
-          replaceDefaultImage(input: {
+          saveDefaultImageForWebsite(input: {
```

Comment:
The parameters changed here too.  
The new parameters are:  itemId, defaultFilePath, and objectFileGroupId

Reply:
thanks didn't see that the inputs changed